libs/vkd3d/command.c

```diff
@@ -8983,12 +9012,20 @@ static void STDMETHODCALLTYPE d3d12_command_list_CopyTextureRegion(d3d12_command
                     subres = &copy_info.copy.buffer_image.imageSubresource;
                     other_subres = &other_info->copy.buffer_image.imageSubresource;
                     assert(subres->layerCount == 1 && other_subres->layerCount == 1);
-                    alias = vk_image_copy_subresource_equals(copy_info.dst.pResource, subres,
+                    alias = vk_image_copy_subresource_overlaps(copy_info.dst.pResource, subres,
                             other_info->dst.pResource, other_subres);
                     break;
                 case VKD3D_BATCH_TYPE_COPY_IMAGE:
+                    if (vk_image_copy_subresource_fuse_aspects(
```

Comment:
Staring a bit at this, I think there's a potential issue if we fuse, then later find there was an alias. Need a small rethink.

Reply:
Probably need to decouple the fuse check from actually modifying the aspect mask?